app/assets/javascripts/core/posts_controller.coffee

```diff
@@ -0,0 +1,50 @@
+#= require './view_controller'
+
+class Spaces.PostsController extends Spaces.ViewController
+
+	initialize: ->
+		($ 'body').on 'tagsUpdated', () ->
+			updateRelevantPosts()
+		($ 'body').on 'postAdded', (event, data) ->
+			postAdded data
+		($ 'body').on 'postFollowed', (event, data) ->
+			updateFollowDisplay data
+
+	updateRelevantPosts = () ->
```

Comment:
should define these functions as methods on your `Spaces.PostsController` class rather than as floating functions in your file. You would do this like `updateRelevantPosts: ->` with a colon instead of `=`. Then you would call the function from elsewhere in this class (like on line 7 above) using `@updateRelevantPosts()` instead of `updateRelevantPosts()`